code/vision_processing.py

```diff
@@ -10,7 +10,7 @@
 try:
     client = vision.ImageAnnotatorClient()
 except:
-    debug("Google Vision API is not setup, please run /setup")
+    print("Google Vision API is not setup, please run /setup")
```

Comment:
Google vision API is global, should be `add your google credentials file in the /data/google-vision/botator.json file.`

Reply:
Was just a way to make it not mandatory, still works the same way, but doesn't crash if the cred file isn't there